tests/v3_api/test_upgrade.py

```diff
@@ -171,6 +176,14 @@ def test_create_and_validate_ingress_xip_io_wl():
     create_and_validate_ingress_xip_io_wl()
 
 
+def test_rancher_upgrade():
+    upgrade_rancher_server(CATTLE_TEST_URL)
+    time.sleep(300)
```

Comment:
@bmdepesa This sleep is for waiting the user clusters in the server to get to active state ? If so , can we not have this sleep here and instead have it in the jenkin's job before running post upgrade checks on the user clusters?